src/test/java/com/endava/intership/collections/StudentTest.java

```diff
@@ -0,0 +1,84 @@
+package com.endava.intership.collections;
+
+
+import com.endava.internship.collections.Student;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullSource;
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+import java.time.LocalDate;
+
+public class StudentTest {
+
+    private Student A;
```

Comment:
make fields final

Reply:
I can't make field A final because initialization doesn't happen in the constructor